DIMS-Core.Database/DIMS-Core.Database/Procedures/SetUserTaskAsSuccess.sql

```diff
@@ -0,0 +1,15 @@
+﻿CREATE PROCEDURE [dbo].[SetUserTaskAsSuccess]
+	@UserId int,
+	@TaskId int
+AS
+BEGIN
```

Comment:
Why have you added begin and end in the procedure?

Reply:
I have added BEGIN and END in the procedure to separate procedure's body from another code.
Like in tutorial:
[https://metanit.com/sql/sqlserver/11.1.php](url)